src/component/Calculator.js

```diff
@@ -1,67 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-  }
+function Calculator() {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
 
-  handleClick = (e) => {
+  const handleClick = (e) => {
     const num = e.target.textContent;
-    const ans = calculate(this.state, num);
-    this.setState(ans);
-  }
+    const ans = calculate(state, num);
+    setState(ans);
+  };
 
-  render() {
-    const { total, next, operation } = this.state;
-    return (
-      <div className="main">
-        <div className="min-calculator">
-          <div className="min-bar">
-            {total}
-            {operation}
-            {next}
-          </div>
-          <div className="first row">
-            <button type="button" onClick={this.handleClick}>AC</button>
-            <button type="button" onClick={this.handleClick}>+/-</button>
-            <button type="button" onClick={this.handleClick}>%</button>
-            <button type="button" onClick={this.handleClick} className="color">÷</button>
-          </div>
-          <div className="2nd row">
-            <button type="button" onClick={this.handleClick}>7</button>
-            <button type="button" onClick={this.handleClick}>8</button>
-            <button type="button" onClick={this.handleClick}>9</button>
-            <button type="button" onClick={this.handleClick} className="color">×</button>
-          </div>
```

Comment:
- [ ] Your multiplication operation is not working. Kindly fix this to make your calculator fully usable :+1: